packages/api/amplify_api/test/query_predicate_graphql_filter_test.dart

```diff
@@ -209,5 +211,17 @@ void main() {
       _testQueryPredicateTranslation(queryPredicate, expectedFilter,
           modelType: Post.classType);
     });
+
+    test('query with enum should serialize to string', () {
+      // Note: enums are not actually in the codegen model. Nonetheless, we
+      // expect this predicate to translate as follows.
+      final queryPredicate = Post.TITLE.eq(Size.MEDIUM);
+      final expectedFilter = {
+        'title': {'eq': 'MEDIUM'}
```

Comment:
nit: use `describeEnum` as well to ensure the same behavior

Reply:
good call, will change